_codux/ui-kit/buttons/buttons.board.tsx

```diff
@@ -0,0 +1,141 @@
+import { createBoard, Variant } from '@wixc3/react-board';
+import { CartIcon, FacebookIcon, PinterestIcon, WhatsAppIcon } from '~/components/icons';
+import { LabelWithArrow } from '~/components/label-with-arrow/label-with-arrow';
+import { TrashIcon } from '../../../src/components/icons/trash-icon';
+import { LockIcon } from '../../../src/components/icons/lock-icon';
+import { ErrorIcon } from '../../../src/components/icons/error-icon';
+import { MenuIcon } from '../../../src/components/icons/menu-icon';
+import { ChevronRightIcon } from '../../../src/components/icons/chevron-right-icon';
+import { ArrowRightIcon } from '../../../src/components/icons/arrow-right-icon';
+import { Kit } from '../ui-kit-utils/kit';
+
+import styles from './buttons.board.module.scss';
+
+export default createBoard({
+    name: 'Buttons',
+    Board: () => (
+        <Kit category="Core Components" title="Buttons">
+            <Kit.Section title="Buttons" className={styles.demoWidth}>
+                <Kit.Item>
+                    <Variant name="Primary Button">
+                        <button className="button primaryButton">Add to Cart</button>
+                    </Variant>
+                    <Kit.Description>Primary Button</Kit.Description>
+                </Kit.Item>
+                <Kit.Item>
+                    <Variant name="Primary Muted Button">
+                        <button className="button mutedPrimaryButton">Add to Cart</button>
+                    </Variant>
+                    <Kit.Description>Primary Muted Button</Kit.Description>
+                </Kit.Item>
+                <Kit.Item>
+                    <Variant name="Secondary Button">
+                        <button className="button secondary">Add to Cart</button>
+                    </Variant>
+                    <Kit.Description>Secondary Button</Kit.Description>
+                </Kit.Item>
+                <Kit.Item>
+                    <Variant name="Label With Arrow">
+                        <LabelWithArrow>Shop Now</LabelWithArrow>
+                    </Variant>
+                    <Kit.Description>Label With Arrow</Kit.Description>
+                </Kit.Item>
+            </Kit.Section>
+
+            <Kit.Section title="Icons">
+                <div className={styles.itemsGroup}>
+                    <Kit.Item>
+                        <div className={styles.icon}>
```

Comment:
I guess we can put className on the <Kit.Item>?

Reply:
It has `className`, but I need additional `div`. 
Item here is `flex` column, `div.icon` is the wrapper where I center an icon.